tests/ColorTests/FunctionalTest/color_ft.cpp

```diff
@@ -642,8 +650,15 @@ TEST_P(color_control_test, control_test)
 
     if (as.command == K4A_COLOR_CONTROL_EXPOSURE_TIME_ABSOLUTE)
     {
+#ifdef _WIN32
+        // Windows only support exposure time value as log base 2 seconds.
```

Comment:
We should talk about this, what will it take to remove #ifdef's in the code? This means to have  different behavior for different platforms, and I would prefer to avoid that. This also means we have to document the behavior too.

Reply:
I will think how we can eliminate #ifdef for this.
And documentation bug is opened #240

Reply:
The goal is that this enhancement doesn't result in more complicated documentation.

---
In reply to: [280930242](https://github.com/Microsoft/Azure-Kinect-Sensor-SDK/pull/319#discussion_r280930242) [](ancestors = 280930242)

Reply:
I think this #ifdef does not make more complicated documentation, but it support #240 documentation for better understanding.